docs/upgrade/2.11.1_to_2.12.0.rst

```diff
@@ -0,0 +1,90 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.11.1 to 2.12.0
+=============================
+
+Migrating to Ubuntu 24.04 (Noble)
+---------------------------------
+
+The SecureDrop 2.12.0 release provides the foundation necessary to safely upgrade your SecureDrop Servers to Ubuntu 24.04 (Noble), which is necessary due to the upcoming end-of-life for Ubuntu 20.04 (Focal).
+
+Administrators have two options, on the following timeline:
+
+* **semiautomated, through April 15, 2025:** Administrators can manually trigger the upgrade and observe the process.
+* **fully automated, after April 15, 2025:** The SecureDrop team will push an update in mid- to late-April that automatically begins the upgrade process on all servers.
+
+To determine which option is best for you, and to learn more about how the upgrade works, please review the :doc:`Ubuntu 24.04 (Noble) migration guide <../admin/maintenance/noble_migration>` at your earliest convenience.
+    
+Update Servers to SecureDrop 2.12.0
+------------------------------------
+
+Servers running Ubuntu 20.04 will be updated to the latest version of SecureDrop
```

Comment:
I like your clarification that this 2.12.0 upgrade is not the same as the noble upgrade - small suggestion to make it clearer by removing any mention of the Ubuntu version in this first sentence.

```suggestion
Servers will be updated to the latest version of SecureDrop
```